ui/raidboss/data/02-arr/trial/cape_westwind.txt

```diff
@@ -1,6 +1,8 @@
 ### Cape Westwind
 # -ii 0A 2CD 2CE 194 14 -p 474:204 478:610
 
+# Trial removed in Patch 6.1
```

Comment:
IIRC, the only reason we keep old content around is if there's an int'l version that might still use it, right?  And I think all the int'l versions are past 6.1?

Reply:
We might want to keep this one around as long as the Timeline Guide references it.

Reply:
If we did want to keep it for reference purposes (and don't want to do a rewrite of the Timeline Guide right away), we can remove the triggers and timeline from the data folder and the manifest, and readd the timeline into the docs folder (call it sample_timeline.txt or something like that).

Reply:
We can always link to it in git history by commit hash too.  No need to keep it around.

Honestly, that timeline guide needs to be rewritten anyway, so <_<

Reply:
> We can always link to it in git history by commit hash too. No need to keep it around.
> 
> Honestly, that timeline guide needs to be rewritten anyway, so <_<

Done.  I was already attempting a (modest) enhancement to the timeline guide based on these changes and committed it before I saw your comment.  It should be in the latest two commits, if you have a chance to look.  Otherwise, will work through these most recent comments and then move on to sb/shb/ew.